crates/aptos/src/common/types.rs

```diff
@@ -919,6 +919,10 @@ pub struct MovePackageDir {
     /// this for local development.
     #[clap(long)]
     pub(crate) skip_fetch_latest_git_deps: bool,
+
+    /// Specify the version of the bytecode the compiler is going to emit.
+    #[clap(long)]
+    pub bytecode_version: Option<u32>,
```

Comment:
This is related?

Reply:
Yes. By default, this flag is not set and the cli will tell the move compiler to use bytecode version 5 so to maintain compatibility with our current testnet and mainnet. The user can however, opt into version 6 using this flag if they want to try out the new integer types.